File: 1768-merge-strings-alternately/1768-merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        int i=0;
        int j=0;
        string str="";
        while(i<word1.size() && j<word2.size()){
            str+=(word1[i]);
            str+=(word2[j]);
            i++;j++;
        }
        while(i<word1.size()){
            str+=(word1[i]);i++;
        }
        while(j<word2.size()){
            str+=(word2[j]);j++;
        }
        return str;
        
    }
};

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        ans=""
        i=0
        j=0
        while i<len(word1) and j<len(word2):
            ans=ans+word1[i]
            i=i+1
            ans=ans+word2[j]
            j=j+1
        if len(word1)>len(word2):
            ans=ans+word1[len(word2):]
            return ans
        if len(word1)<len(word2):
            ans=ans+word2[len(word1):]
            return ans
        return ans
        